overlord/servicestate/export_test.go

```diff
@@ -19,6 +19,16 @@
 
 package servicestate
 
+import "time"
+
 var (
 	UpdateSnapstateServices = updateSnapstateServices
 )
+
+func MockBootTime(f func() (time.Time, error)) (restore func()) {
```

Comment:
isn't mocking why MockCommand enough? it would safe as one test

Reply:
that's fair, I can remove the duplication here, tbh I forgot about MockCommand initially so I wrote the test without it and then remembered that it would be good to test the real getBootTime function so I just added another test with MockCommand rather than think deeply about the fact that the tests are pretty much the same

Reply:
done